pkg/operator/ceph/cluster/mon/config.go

```diff
@@ -111,6 +111,11 @@ func CreateOrLoadClusterInfo(context *clusterd.Context, namespace string, ownerR
 		logger.Debugf("found existing monitor secrets for cluster %s", clusterInfo.Name)
 	}
 
+	// ensure the csi secret exists
```

Comment:
@travisn JFYI with the current implementation we are creating the CSI secrets even if CSI is disabled